replication-api-impl/src/main/java/org/codice/ditto/replication/api/impl/ReplicatorImpl.java

```diff
@@ -106,6 +106,7 @@ public void init() {
             } catch (InterruptedException e) {
```

Comment:
✏️ The above block nested inside the `execute()` should be moved to a private method